test/jdk/javax/print/PrintServiceLookup/CountPrintServices.java

```diff
@@ -25,33 +25,35 @@
 import java.io.InputStreamReader;
 import javax.print.PrintService;
 import javax.print.PrintServiceLookup;
-import javax.print.attribute.AttributeSet;
-import javax.print.attribute.HashAttributeSet;
-import javax.print.attribute.standard.PrinterName;
+
+import jtreg.SkippedException;
 
 /*
  * @test
  * @bug 8032693
  * @key printer
+ * @library /test/lib/
+ * @requires (os.family == "linux")
  * @summary Test that lpstat and JDK agree whether there are printers.
  */
 public class CountPrintServices {
 
   public static void main(String[] args) throws Exception {
-    String os = System.getProperty("os.name").toLowerCase();
-    System.out.println("OS is " + os);
-    if (!os.equals("linux")) {
-        System.out.println("Linux specific test. No need to continue");
-        return;
-    }
     PrintService services[] =
         PrintServiceLookup.lookupPrintServices(null, null);
     if (services.length > 0) {
        System.out.println("Services found. No need to test further.");
        return;
     }
     String[] lpcmd = { "lpstat", "-a" };
-    Process proc = Runtime.getRuntime().exec(lpcmd);
+    Process proc = null;
+    try {
+        proc = Runtime.getRuntime().exec(lpcmd);
+    } catch (java.io.IOException e) {
+        if(e.getMessage().contains("No such file or directory")) {
```

Comment:
```suggestion
        if (e.getMessage().contains("No such file or directory")) {
```
A space after the `if` keyword.

Reply:
Thanks, fixed.